src/modules/member-friendship/listeners/member-friendships.listeners.ts

```diff
@@ -0,0 +1,97 @@
+import { Injectable } from '@nestjs/common';
+import { OnEvent } from '@nestjs/event-emitter';
+import { PrismaService } from '@src/modules/core/database/prisma/prisma.service';
+import { NotificationService } from '@src/modules/core/notification/services/notification.service';
+
+export const MEMBER_FOLLOW_CREATE = 'memberFollow.create';
+export const MEMBER_FOLLOW_DELETE = 'memberFollow.delete';
+
+@Injectable()
+export class MemberFriendshipsListeners {
+  constructor(
+    private readonly prismaService: PrismaService,
+    private readonly notificationService: NotificationService,
+  ) {}
+
+  @OnEvent(MEMBER_FOLLOW_CREATE)
+  handleMemberFollowCreateEvent(toMemberId: number, fromMemberId: number) {
+    this.prismaService
+      .$transaction([
+        this.prismaService.memberStatistics.update({
+          data: {
+            followerCount: {
+              increment: 1,
+            },
+          },
+          where: {
+            memberId: fromMemberId,
+          },
+        }),
+        this.prismaService.memberStatistics.update({
+          data: {
+            followingCount: {
+              increment: 1,
+            },
+          },
+          where: {
+            memberId: toMemberId,
+          },
+        }),
+      ])
+      .catch((e) => {
+        this.notificationService
+          .warning({
+            description: 'handleMemberFollowCreateEvent 중 에러',
+            body: {
+              toMemberId,
+              fromMemberId,
+            },
+            stack: e.stack,
+          })
+          .catch((e) => {
+            console.error(e);
+          });
+      });
+  }
+
+  @OnEvent(MEMBER_FOLLOW_DELETE)
+  handleMemberFollowDeleteEvent(toMemberId: number, fromMemberId: number) {
+    this.prismaService
+      .$transaction([
+        this.prismaService.memberStatistics.update({
+          data: {
+            followerCount: {
+              decrement: 1,
+            },
+          },
+          where: {
+            memberId: fromMemberId,
```

Comment:
```suggestion
            memberId: toMemberId,
```
팔로워 수(나를 팔로우 하는 사람의 수)는 toMemberId(팔로우 신청을 받은 멤버)
가 증가되어야 하는 것 아닌가요?

제가 석호님께 편지를 작성을 하는 상황이라고 가정하고 편지를 작성할 때 
일반적으로 "to 석호 from 수형"으로 작성하는 것으로 알고 있습니다

Reply:
수정하도록 하겠습니다.